lib/yabeda/sidekiq/client_middleware.rb

```diff
@@ -5,8 +5,14 @@ module Sidekiq
     # Client middleware to count number of enqueued jobs
     class ClientMiddleware
       def call(worker, job, queue, _redis_pool)
-        labels = Yabeda::Sidekiq.labelize(worker, job, queue)
+        labels = Yabeda::Sidekiq.labelize(worker, job, job["queue"] || queue)
         Yabeda.sidekiq_jobs_enqueued_total.increment(labels)
+
+        if job["queue"] != queue
```

Comment:
I may be overcautious here, but should we check for `job["queue"]` presence also here?
```suggestion
        if job["queue"] && job["queue"] != queue
```